web/src/components/Section.js

```diff
@@ -21,7 +21,7 @@ const Section = ({ children, id, isNumbered, resource }) => {
   const helptext = t([resource, 'helpText'], { defaultValue: false });
 
   return (
-    <section id={id} className={isNumbered && 'numbered-section'}>
+    <section id={id} className={isNumbered ? 'numbered-section' : ''}>
```

Comment:
This was my suggestion, but it turns out React complains about `className` having a boolean value.  My mistake!

Reply:
Thanks for fixing it!